File: Python_CS-150/Caesar_Cipher.py
# /**
#  *  A simple Caesar Cipher encrypter and decrypter.
#  *
#  * @author Marshall Livingston <br>
#  *         mltechi@rams.colostate.edu <br>
#  *         Computer Science Department <br>
#  *         Colorado State University
#  * @version 201990
#  */

class Caesar_Cipher:

    def __init__(self, message):
        self.message = message.lower()
        # add here as you go

    def shift(self, shift):
        message = self.message
        i = 0
        while i < len(message):
            i += 1
            for ch in message:
                print(ord(ch))
            break
        


test_1 = Caesar_Cipher('HELLO')
print(test_1.shift(2))

File: CS-150/java/CaesarCipher.java
import java.util.Scanner;

/**
 *  A simple Caesar Cipher encrypter and decrypter.
 *
 * @author Marshall Livingston <br>
 *         mltechi@rams.colostate.edu <br>
 *         Computer Science Department <br>
 *         Colorado State University
 * @version 201990
 */
public class CaesarCipher {

   
  static void shift(int shift, String message) {        // This shift method takes 3 arguments: a char, an int, and a String of which to encrypt with ROT cipher ) 
    message = message.toLowerCase(); 
    for (int i = 0; i < message.length(); ++i) {
      char newChar = message.charAt(i);
      if (newChar >= (char)0 && newChar <= (char)127) {
        newChar = (char)(newChar + shift);
        System.out.print((char)newChar);
      } else {
          System.out.print("Nice job, you broke it.");
        break;
        
      }
    }
  }

  static void unShift(int shift, String message) {
    message = message.toLowerCase();
    for (int i = 0; i < message.length(); ++i) {
      char newChar = message.charAt(i);
      if (newChar >= (char)0 && newChar <= (char)127) {
        newChar = (char)(newChar - shift);
        System.out.print((char)newChar);
      } else {
          System.out.print("Nice job, you broke it.");
        break;
      }    
    }
  }

  public static void main(String[] args) {
    Scanner scnr = new Scanner(System.in);
    shift(2, scnr.nextLine());
    System.out.println();
    unShift(2, scnr.nextLine());
    scnr.close();
  }
}
